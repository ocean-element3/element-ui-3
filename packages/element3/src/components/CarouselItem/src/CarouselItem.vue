<template>
  <div class="el-carousel__item">
    <div class="el-carousel__mask"></div>
    <slot></slot>
  </div>
</template>

<script>
export default {
  name: 'ElCarouselItem',
  props: {
    label: Number
  }
}
</script>
