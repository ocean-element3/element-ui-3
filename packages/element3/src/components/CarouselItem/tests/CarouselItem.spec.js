import CarouselItem from '../src/CarouselItem.vue'
import { mount } from '@vue/test-utils'

describe('Carousel.vue', () => {
  it('snapshot', () => {
    const wrapper = mount(CarouselItem)
    expect(wrapper.element).toMatchSnapshot()
  })
})
