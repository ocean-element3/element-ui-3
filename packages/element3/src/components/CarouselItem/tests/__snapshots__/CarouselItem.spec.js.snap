// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel.vue snapshot 1`] = `
<div
  class="el-carousel__item"
  style="display: none;"
/>
`;
